Mapify/Map/MapLifeCycle.cs

```diff
@@ -50,8 +50,12 @@ public static IEnumerator LoadMap(BasicMapInfo basicMapInfo)
             loadingInfo.UpdateLoadingStatus(loadingMapLogMsg, 0);
             yield return null;
 
-            // Load asset bundles
             string mapDir = Maps.GetDirectory(basicMapInfo);
+
+            // Register translations
+            Locale.LoadMapCSV(mapDir+"/../");
```

Comment:
Is this correct? Shouldn't it be passing in the map directory?

Reply:
This is correct. The file which contains the translations (`locale.csv` ) is meant to be placed next to the `info.json`, which is one level up from the MapInfo file.